### PR TITLE
Move etcd v2 code to rancher-etcd

### PR DESCRIPTION
**Pull request by galal-hussein**

### CODE REVIEW

File: package/Dockerfile [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+  apt-get clean && \
   wget -q -O /opt/rancher/giddyup https://github.com/rancher/giddyup/releases/download/v0.18.0/giddyup && \
   chmod +x /opt/rancher/giddyup
 
 RUN \
-  wget -q -O - https://github.com/coreos/etcd/releases/download/v3.0.17/etcd-v3.0.17-linux-amd64.tar.gz | tar xzf - -C /tmp && \
+  wget -q -O - https://github.com/coreos/etcd/releases/download/v2.3.7/etcd-v2.3.7-linux-amd64.tar.gz | tar xzf - -C /tmp && \
   mv /tmp/etcd-*/etcd /opt/rancher/etcd && \
   mv /tmp/etcd-*/etcdctl /opt/rancher/etcdctl && \
   rm -rf /tmp/etcd-* && rm -f /etcd-*.tar.gz
```
Comment (leodotcloud) [inline package/Dockerfile]: Changes in this Dockerfile looks good.

**Comment by leodotcloud** [comment]
@galal-hussein can you please quickly test with the image built from this repo? Once that passes, we are good to merge and release.

**Comment by galal-hussein** [comment]
@leodotcloud I tested the image built from this repo, seems to be working correctly.

**Comment by leodotcloud** [comment]
LGTM

**Pull request merged**